infra/feast-operator/test/testdata/feast_integration_test_crs/v1alpha1_remote_registry_featurestore.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: feast.dev/v1alpha1
+kind: FeatureStore
+metadata:
+  name: simple-feast-remote-setup
+spec:
+  feastProject: my_project
+  services:
+    onlineStore:
+      image: 'localhost/feastdev/feature-server:dev'
+    offlineStore:
+      image: 'localhost/feastdev/feature-server:dev'
+    registry:
+      remote:
+        hostname: 'feast-simple-feast-setup-registry.default.svc.cluster.local:80'
```

let's test the `remote.feastRef` functionality here instead -
```suggestion
        feastRef:
          name: simple-feast-setup
          namespace: remote-registry
```

---

[Fixed it](https://github.com/feast-dev/feast/pull/4822/commits/60c10b1783f9d5e0c18dbe4d3197b5a64afe7bff)